packages/@webex/plugin-meetings/src/meetings/index.ts

```diff
@@ -1369,11 +1369,11 @@ export default class Meetings extends WebexPlugin {
         const meetingsCollection = this.meetingCollection.getAll();
 
         if (Object.keys(meetingsCollection).length > 0) {
-          // Some time the mercury event is missed after mercury reconnect
-          // if sync returns no locus then clear all the meetings
+          // Sometimes the mercury events are lost after mercury reconnect
+          // Remove any Locus meetings that are not returned by Locus in the sync
```

Comment:
nitpick: Strictly speaking, you aren't removing _any_ meetings not returned by Locus, but those that previously had a locus url and are no longer active. Not sure of a succinct way to phrase that though for the comment though!